tests/container_test.py

```diff
@@ -29,15 +29,15 @@ def test_container_count(dockerc):
     """Verify the test composition and container."""
     # stopped parameter allows non-running containers in results
```

Comment:
I just noticed that this comment should be changed up in [skeleton-docker](https://github.com/cisagov/skeleton-docker/blob/develop/tests/container_test.py#L23) since we no longer include the `stopped` parameter.

Reply:
Please see cisagov/skeleton-docker#168.

Reply:
Thanks!